_posts/2019-12-17-solidity-0.6.0-release-announcement.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.6.0 Release Announcement'
+date: '2019-12-17'
+author: Solidity Team
+category: Releases
+---
+
+This is a major breaking release of the Solidity compiler and language. Changes include explicit virtual and override keywords in inheritance, support for try/catch, splitting the fallback function into a receive Ether function and an actual fallback function and limitations on how the length of an array can be changed, among others. For a detailed explanation, please see the [documentation](https://solidity.readthedocs.io/en/latest/060-breaking-changes.html) or refer to the list below that shows every single change.
+
+From this release on, ABIEncoderV2 is not considered experimental any more, but you still have to activate it through the pragma.
+
+Furthermore, the Yul optimizer is automatically activated together with the regular optimizer, but you can still disable it through the detailed optimizer settings.
+
+**Breaking changes:**
```

Comment:
```suggestion
**Breaking Changes:**
```

Would capitalize the `changes`since we're also doing so with the others like `New Features`.

Reply:
This is coming directly from the release notes page. Do we want to change them here?

Reply:
Oh I see, I don't know. I just saw the inconsistency and wanted to point it out.

Reply:
Updated. I guess we can change these once they are extracted, they don't need to be a 100% match :)